### DISCUSSION TITLE
api/v1/chats/all/tags 400 (Bad Request)

### DISCUSSION THREAD

**Discussion by PhantomeAlighieri** [open-webui/open-webui#6448] [Issues]
After the last two updates when logging in I'm faced with a tag issue.

GET http://192.168.1.140:3000/api/v1/chats/all/tags 400 (Bad Request)

<html>
<body>

A | @ | index.ts:344
-- | -- | --
  | (anonymous) | @ | +layout.svelte:120
  | (anonymous) | @ | +layout.svelte:120
  | Promise.then |   |  
  | (anonymous) | @ | +layout.svelte:189
  | await in (anonymous) |   |  
  | ht | @ | root.svelte:23
  | await in Y |   |  
  | (anonymous) | @ | (index):102
  | Promise.then |   |  
  | (anonymous) | @ | (index):101

</body>
</html>
  | (anonymous) | @ | +layout.svelte:218
-- | -- | -- | --
  | Promise.then |   |  
  | (anonymous) | @ | +layout.svelte:189
  | await in (anonymous) |   |  
  | ht | @ | root.svelte:23
  | await in Y |   |  
  | (anonymous) | @ | (index):102
  | Promise.then |   |  
  | (anonymous) | @ | (index):101

</span></div>
</body>
</html>

![image](https://github.com/user-attachments/assets/f9985a90-3b31-4c1d-aab2-9f42b4c994c6)

running on ubuntu server, using docker container.

**Comment by SnowLeopard71** [comment]
I upgraded from 0.3.3 to 0.3.5 of a direct install but skipped the *npm* part (*npm install && npm run build*) and when loading the UI, the request on _/api/v1/chats/tags/all_ was giving a **404 Not Found** in the console and the browser showed a blank page.

After redoing the upgrade without skipping the *npm* commands, everything works as it should without any errors in the console.  Maybe this info is useful.

**Reply by PhantomeAlighieri** [reply]
I had it installed as a docker container.  I did several upgrades, rollbacks, and so on--nothing worked.  I just ended up deleting the virtual machine I run it through on my ESXI server, and then set up the docker container again.  Everything works now--I did loose my conversation history in the process.  But, now I have a fresh VM, and a fresh open-webui setup for use.
